jscomp/syntax/src/res_printer.ml

```diff
@@ -4146,7 +4146,13 @@ and print_pexp_apply ~state expr cmt_tbl =
     let partial, attrs = ParsetreeViewer.process_partial_app_attribute attrs in
     let args =
       if partial then
-        let dummy = Ast_helper.Exp.constant (Ast_helper.Const.int 0) in
+        let loc =
+          {Asttypes.txt = "res.partial"; Asttypes.loc = expr.pexp_loc}
```

Comment:
For the dummy attribute, I added `res.partial` so it can be pattern matched with the attribute rather than the label.

Reply:
Looking at previous code, it doesn't look like the choice of the `expression` here matters.